activerecord/lib/active_record/connection_adapters/trilogy_adapter.rb

```diff
@@ -41,6 +39,10 @@ class TrilogyAdapter < AbstractMysqlAdapter
       ER_BAD_DB_ERROR = 1049
       ER_DBACCESS_DENIED_ERROR = 1044
       ER_ACCESS_DENIED_ERROR = 1045
+      ER_SERVER_SHUTDOWN = 1053
+      CR_SERVER_LOST = 2013
+      CR_SERVER_LOST_EXTENDED = 2055
+      CR_SERVER_GONE_ERROR = 2006
```

Comment:
These come from libmysqlclient, so I suspect we won't need them in the Trilogy adapter?

Reply:
Yup, good catch.